src/default_parameters.jl

```diff
@@ -59,23 +59,32 @@
     RKs::Int=3                          # Number of stages for SSPRK2
     RKn::Int=5                          # n^2 = s = Number of stages  for SSPRK3
     cfl::Real=0.9                       # CFL number (1.0 recommended for RK4, 0.6 for RK3)
-    Ndays::Real=200.0                   # number of days to integrate for
+    Ndays::Real=5                       # number of days to integrate for
     nstep_diff::Int=1                   # diffusive part every nstep_diff time steps.
     nstep_advcor::Int=0                 # advection and coriolis update every nstep_advcor time steps.
                                         # 0 means it is included in every RK4 substep
     compensated::Bool=false             # Compensated summation in the time integration?
 
     # BOUNDARY CONDITION OPTIONS
-    bc::String="periodic"               # "periodic" or anything else for nonperiodic
-    α::Real=2.                          # lateral boundary condition parameter
+    bc::String="nonperiodic"            # "periodic" or anything else for nonperiodic
+    α::Real=0.                          # lateral boundary condition parameter
                                         # 0 free-slip, 0<α<2 partial-slip, 2 no-slip
 
+    # PARAMETERS FOR ADJOINT METHOD
+    data_steps::StepRange{Int,Int} = 0:1:0      # Timesteps where data exists
+    data::Array{Float32, 1} = [0.]              # model data
+    J::Float64 = 0.                             # Placeholder for cost function evaluation
+    j::Int = 1                                  # For keeping track of the entry in data
+
+    # CHECKPOINTING VARIABLES
+    i::Int = 0                                  # Placeholder for current timestep, needed for Checkpointing.jl
```

Comment:
Add these only when adding Enzyme/Checkpointing stuff?

Reply:
Yes, or more specfically cost-function related values, mostly because I compute a model-data misfit as I integrate (if I have data at step `t`, then I enter an if statement that adds the needed amount to `J`)

Reply:
Remove then and add when also adding Enzyme/Checkpointing?

Reply:
Do you mean to define a separate `Parameters` structure that is used only when using Enzyme/Checkpointing?

Reply:
Nah, it doesn't look like anything big, although "model data" isn't actually data?? Parameters shouldn't change during the model integration really....

Reply:
Yeah that's all fine then. In SpeedyWeather we're a bit more strict on these kinds of logics of what cannot depend on what but here it's more of a free for all anyway, so leave it in, take it out, as you like!